include/common/list.h

```diff
@@ -0,0 +1,19 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
+#include <wayland-server-core.h>
+
+/**
+ * wl_list_append() - add a new element to the end of a list
+ * @list: list head to add it before
+ * @elm: new element to be added (link of the containing struct to be precise)
+ *
+ * Note: In labwc, most lists are queues where we want to add new elements to
+ * the end of the list. As wl_list_insert() adds elements at the front of the
+ * list (like a stack) - without this helper-functions - users have to add call
```

Comment:
`s/add//`

nit: `s/users/we/` or `s/users have/one has/` ?

Reply:
Done